src/app/authentication/forgot-password/forgot-password.component.ts

```diff
@@ -36,25 +36,24 @@ export class ForgotPasswordComponent {
                         return this.popupService.new({
                             type: 'error',
                             title: 'Not found',
-                            text: `User doesn't exist on the platform`
+                            // tslint:disable-next-line
```

Comment:
I suppose that you can remove this now.

Reply:
Ah.. what a mess.. 

OK. After some investigating about it, use backticks and remove tslint disable.

In other words, revert back to  "backtick - `User doesn't exist on the platform` - backtick"

Reply:
Lint acting silly 😄 
Sure thing